src/fu-tool.c

```diff
@@ -800,7 +800,7 @@ fu_util_firmware_read (FuUtilPrivate *priv, gchar **values, GError **error)
 			  G_CALLBACK (fu_util_update_device_changed_cb), priv);
 
 	/* dump firmware */
-	blob_fw = fu_engine_firmware_read (priv->engine, device, priv->flags, error);
+	blob_fw = fu_engine_firmware_dump (priv->engine, device, priv->flags, error);
```

Comment:
does the fu-tool command want to be re-named now?

Reply:
I'm not sure. I think to the end user it's still "Logically correct" in that it's reading the firmware from the device, it's just fwupd that cares about the distinction between `dump` and `read`. The user isn't going to want to read the firmware and do `sha1sum` on it manually to get the LVFS checksum, I don't think.

Reply:
I guess since there is also a parse command, I would think renaming to dump makes sense.  You can conceivable "dump" and then bring it into "parse".

Reply:
>  You can conceivable "dump" and then bring it into "parse".

Yes, that makes much more sense. Fixed, thanks.